libpod/images/image_data.go

```diff
@@ -88,7 +88,6 @@ func GetData(store storage.Store, name string) (*Data, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "error reading image %q", img.ID)
 	}
-	defer imgRef.Close()
```

Comment:
Did something happen in c/image so that we don't need to close these anymore?

Reply:
yeah c/image has changed quite a bit, and the Close method for types.Image no longer exists.